### DISCUSSION TITLE
How to configurate auto include to use <> instead of ""?

### DISCUSSION THREAD

**Discussion by FlyinCow** [clangd/clangd#2257] [Q&A] [answered]
How to configurate auto include to use <> instead of ""? I'm using vscode clangd extension, I don't know if this happens on other situations (nvim, etc.).

**Comment by HighCommander4** [comment, marked as answer]
We don't currently have a config option for this, but there is a work in progress PR to add one: https://github.com/llvm/llvm-project/pull/67749.
